src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -161,6 +166,14 @@ private void LoadComponents()
 
             LoadAnalyzerNodeComponents();
 
+            // Ensure the nuget package services are initialized after we've loaded
```

Comment:
ah, sorry, git collapsed region and it looked like code belong to initialize.

if possible, importing and running from background thread will be even better.